app/models/abuse_report.rb

```diff
@@ -46,6 +46,10 @@ def akismet_attributes
   # for "/works/123"
   before_validation :clean_url, on: :create
   def clean_url
+    if url =~ %r{(chapters/\d+)} && url !~ %r{(works/\d+)}
```

Comment:
I think adding the work ID is different enough from cleaning that this should be its own `before_validation`.  Can it go after `clean_url` or does it need to come before?

Reply:
> Thank you for working on this -- I hope exams went well!

Cheers, still waiting for grades but should be all good.

>I think adding the work ID is different enough from cleaning that this should be its own before_validation. Can it go after clean_url or does it need to come before?

Needs to come before as clean_url does stuff to URL's with /works in them. I'll make it a seperate function tho this is my first time even looking at Ruby so the whole before_validation stuff is screwing with my brain